p4-fusion/commands/change_list.cc

```diff
@@ -75,7 +74,6 @@ void ChangeList::StartDownload(P4API* p4, const BranchSet& branchSet, const int&
 		    { return downloadPrepared->load(); });
 	}
 
-	std::shared_ptr<std::vector<std::string>> printBatchFiles = std::make_shared<std::vector<std::string>>();
```

Comment:
Suggestion: Remove the `shared_ptr` in the vector below. The lifetime of the vector is very clear; it is a local variable that will be destroyed at the end of this function.

AIUI, there is nothing async going on where the vector will be referenced by some other thread that was launched, and that reference may be alive while this function exits.